src/photos.cs

```diff
@@ -416,6 +427,18 @@ interface PHObjectChangeDetails {
 		bool ObjectWasDeleted { get; }
 	}
 
+	[iOS (13,0)]
+	[TV (13,0)]
+	[Mac (10,15, onlyOn64 : true)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface PHChangeRequest {
+
+		[Internal]
```

Comment:
Why is this internal?

Reply:
I will make a comment of this too, but the reasoning is a little confusing to me. I remember I could not build the file and @dalexsoto may remember, but there is a file src/Photos/PHAssetChangeRequest.cs that has an Obsolete attribute in the PHAssetChangeRequest empty constructor and we deemed that is what was causing the build issue. However, now I am confused because this class is PHChangeRequest and not PHAssetChangeRequest. I can talk to Alex about this tomorrow and get back to you with more clarification.

Reply:
So below is the code from a file that shows that PHAssetChangeRequest disables creating the no argument constructor and since the base class of PHAssetChangeRequest changed from NSObject to the newly added PHChangeRequest, we thought this would be the way to go. Do you have a different suggestion to handle that?

https://github.com/xamarin/xamarin-macios/blob/c99ad292b2e357f4f2acb083e03281a323da44a2/src/Photos/PHAssetChangeRequest.cs#L7-L15

Reply:
I think I see: you need the default constructor in the base class so that the derived class' default constructor can call it.

I think it would be better to add this in a manually written source file, like the constructor is in PHAssetChangeRequest.

i.e. in the `src/Photos/PHChangeRequest` file (create it if it doesn't exist):

```csharp
public partial class PHChangeRequest {
#if !XAMCORE_3_0
    // This constructor is required for the default constructor in PHAssetChangeRequest to compile.
    internal PHChangeRequest ()
    {
    }
#endif
}
```

this should work around the compiler error, and it doesn't require generating a binding.